DuckDuckGo/FireButtonAnimationTriggers.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  FireButtonAnimationTriggers.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import Core
+
+extension FireButton {
+    
+    static let calendarUTC = {
+        var calendarUTC = Calendar(identifier: .gregorian)
+        calendarUTC.timeZone = TimeZone(identifier: "UTC")!
+        return calendarUTC
+    }()
+    
+    public static var wasFireButtonEverTapped: Bool {
+        AppUserDefaults().wasFireButtonEverTapped ? true : canAssumeFireButtonWasUsedBasedOnInstallDate()
+    }
+    
+    private static func canAssumeFireButtonWasUsedBasedOnInstallDate() -> Bool {
+        guard let installDate = StatisticsUserDefaults().installDate,
+              let featureReleaseDate = calendarUTC.date(from: DateComponents(year: 2023, month: 4, day: 17))
```

Comment:
@miasma13 maybe add a comment with an asana link explaining why this date was picked? Just so it doesn't look random to someone reading the code.